com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/FunctionDescriptorNodeBuilder.cs

```diff
@@ -142,6 +142,9 @@ static TypeDescriptor FallbackTypeResolver(INodeReader userData)
             port.SetField(kHeight, resolvedType.Height);
             port.SetField(kPrecision, resolvedType.Precision);
             port.SetField(kPrimitive, resolvedType.Primitive);
+
+            if (param.Usage == Usage.Static) port.SetField("IsStatic", true);
```

Comment:
Why use `==` here and `is` above?

Reply:
Discussed on Slack: no particular reason, we decided consistently using `is` would be better because we're comparing enums.